gitgud/__main__.py

```diff
@@ -340,6 +349,20 @@ def handle_test(self, args):
         level = self.file_operator.get_level()
         level.test(self.file_operator)
 
+    def handle_solution(self, args):
```

Comment:
This code should be in the `Level` class as the `solution` function. See `goal_str` and `goal`

Reply:
Also see `handle_goal`